data/NinjaArmy.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Joschi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Mostly E-Commerce",
+  "avatar": "https://github.com/NinjaArmy.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/NinjaJoschi",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/NinjaArmy",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "hashnode",
+      "url": "https://ninja-army.hashnode.dev/",
+      "icon": "SiHashnode"
+    }
+  ]
+}
+
```

Comment:
There's an extra line here that should be removed (not 100% sure the CI will fix that automatically).
```suggestion
```

Reply:
We have the Prettier GitHub Action, it will fix the formatting once merged (if the json is valid, but we have a GitHub Action that runs on the pull request to check the json is valid)